ruby/vampires.rb

```diff
@@ -0,0 +1,91 @@
+to_process = 0
+until to_process != 0
+	puts "How many employees will be processed?"
+	to_process = gets.chomp.to_i
+	if to_process == 0
+		puts "Please enter a valid integer above 0"
+	end
+end
+for i in 1..to_process
+	puts "What is your name?"
+	your_name = gets.chomp
+	age = 0
+	until age != 0

```

Comment:
I would love to see these user input gathering loops extracted into their own methods.